File: scholar-web/src/main/webapp/app/model/administration/settings/department/DepartmentSearch.js
Ext.define('scholar.model.administration.settings.department.DepartmentSearch',{
			extend : 'Ext.data.Model',
			idProperty: 'id',
			fields : [ {
				name : 'id',
				type : 'int'
			}, {
				name : 'departmentName',
				type : 'string'
			}, {
				name : 'lastChange',
				type : 'date',
				dateFormat : 'n/j h:ia'
			} ]

		});

File: scholar-business/src/main/java/com/varun/scholar/business/data/entities/administration/settings/DepartmentSearch.java
package com.varun.scholar.business.data.entities.administration.settings;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class DepartmentSearch extends AbstractEntity {
	String departmentName;
	@Temporal(TemporalType.DATE)
	Date lastChange;
}
